sentence.go

```diff
@@ -109,30 +112,41 @@ func Parse(raw string) (Sentence, error) {
 	if err != nil {
 		return nil, err
 	}
-	switch s.Type {
-	case TypeRMC:
-		return newRMC(s)
-	case TypeGGA:
-		return newGGA(s)
-	case TypeGSA:
-		return newGSA(s)
-	case TypeGLL:
-		return newGLL(s)
-	case TypeVTG:
-		return newVTG(s)
-	case TypeZDA:
-		return newZDA(s)
-	case TypePGRME:
-		return newPGRME(s)
-	case TypeGSV:
-		return newGSV(s)
-	case TypeHDT:
-		return newHDT(s)
-	case TypeGNS:
-		return newGNS(s)
-	case TypeTHS:
-		return newTHS(s)
-	default:
-		return nil, fmt.Errorf("nmea: sentence prefix '%s' not supported", s.Prefix())
+
+	if s.Raw[0] == SentenceStartNormal[0] {
+		switch s.Type {
+		case TypeRMC:
+			return newRMC(s)
+		case TypeGGA:
+			return newGGA(s)
+		case TypeGSA:
+			return newGSA(s)
+		case TypeGLL:
+			return newGLL(s)
+		case TypeVTG:
+			return newVTG(s)
+		case TypeZDA:
+			return newZDA(s)
+		case TypePGRME:
+			return newPGRME(s)
+		case TypeGSV:
+			return newGSV(s)
+		case TypeHDT:
+			return newHDT(s)
+		case TypeGNS:
+			return newGNS(s)
+		case TypeTHS:
+			return newTHS(s)
+		}
+	} else if s.Raw[0] == SentenceStartEncapsulated[0] {
+		switch s.Type {
+		case TypeVDM:
+			return newVDMVDO(s)
+		case TypeVDO:
```

Comment:
You can put both prefixes in the same case.

``` go
switch s.Type {
case TypeVDM, TypeVDO:
	return newVDMVDO(s)
}
```

Reply:
Ok